bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVBridgeHandler.java

```diff
@@ -122,13 +124,20 @@ private void initiateConnexion() {
                 String error = uvResponse.getError();
                 if (error == null) {
                     updateStatus(ThingStatus.ONLINE);
+                    updateProperty(KEY_VERIFIED, Boolean.TRUE.toString());
```

Comment:
Does this need to be a property? I guess it means that we have received a well-formed response at least once since binding was initialized. Is that of any particular interest for the user?

Reply:
You're correct on your assumption. I introduced this because sometimes OpenUV fails on a correct API key (apparently both issues seem correlated). Then if the key is reputed correct, the binding will try to reconnect a few minutes later.

Reply:
Perhaps simply an instance variable, assuming that it can/should be reset during initialization?

Reply:
So, not persisting it accross OH restarts ?